pywbem_mock/_baseprovider.py

```diff
@@ -88,6 +76,16 @@ def __init__(self, cimrepository=None, provider_registry=None):
             repository is fully initialized.
```

Comment:
I don't know why it would ever make sense not to know a CIM repository in this class, so I suggest to remove the default `None` from the `cimrepository` init parameter.

Reply:
DONE